pipelines/common/src/main/java/com/google/fhir/analytics/view/ViewApplicator.java

```diff
@@ -550,15 +551,16 @@ public static String getIdString(IIdType id) {
 
   @Getter
   public static class RowElement {
-    private final List<IBase> values;
+    @Nullable private final List<IBase> values;
     private final Column columnInfo;
 
     @Override
     public String toString() {
-      return columnInfo.getName() + ":" + Arrays.toString(values.toArray());
+      return columnInfo.getName() + ":" + Arrays.toString(values != null ? values.toArray() : null);
```

Comment:
Hmm, shouldn't the last `null` be empty string? Did NullAway not give a warning for this?
```suggestion
      return columnInfo.getName() + ":" + Arrays.toString(values != null ? values.toArray() : "");
```